src/utils/removeAllDateSuffix.m

```diff
@@ -2,25 +2,27 @@
 
 function removeAllDateSuffix(rawDir, subjName, sesName)
     %
-    % Short description of what the function does goes here.
+    % Function removes the date suffix in the _events and _stim.tsv files in 
+    % given raw, session and subject folder. And zips the _stim.tsv files.
     %
     % USAGE::
     %
-    %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+    %   removeAllDateSuffix(rawDir, subjName, sesName)
     %
-    % :param argin1: (dimension) obligatory argument. Lorem ipsum dolor sit amet,
-    %                consectetur adipiscing elit. Ut congue nec est ac lacinia.
-    % :type argin1: type
-    % :param argin2: optional argument and its default value. And some of the
-    %               options can be shown in litteral like ``this`` or ``that``.
-    % :type argin2: string
-    % :param argin3: (dimension) optional argument
-    % :type argin3: integer
+    % :param rawDir: obligatory argument. 
+    % :type rawDir: string
+    % :param subjName: obligatory argument. Some of the
```

Comment:
You can remove the `obligatory argument` parts. 

No need to write in full text whether an argument is optional or obligatory but specify that in the `USAGE` section: the optional argument go between square brackets, for example `[optional_arg]`.

In this case all the argument are obligatory.

But we might want later add some code to make the function fail gracefully and give informative messages when the proper inputs are not given.